coalib/processes/Processing.py

```diff
@@ -376,6 +376,7 @@ def instantiate_processes(section,
     filename_list = collect_files(
         glob_list(section.get('files', '')),
         None,
+        line_num=section._get_lines('files'),
```

Comment:
needs to be removed, because you are accessing a semi-private method of an object not in the same component.  i.e. `coalib/processes` shouldnt access `._foo` of an object from `coalib/section` .  (however coalib/section & coalib/settings are almost the same component, so the rule can be bent between those two components, if necessary)